tests/integration/conftest.py

```diff
@@ -79,9 +80,10 @@ def mock_completion(*args, test_name, **kwargs):
     for message in messages:
         message_str += message['content']
     mock_response = get_mock_response(test_name, message_str)
-    assert mock_response is not None, (
-        'Mock response for prompt is not found:\n\n' + message_str
-    )
+    if mock_response is None:
+        print('Mock response for prompt is not found:\n\n' + message_str)
+        print('Exiting...')
+        sys.exit(1)
```

Comment:
I thought the `abort` will raise an AssertionError and abort the program.

Reply:
You would wish so... but it doesn't, because we catch all exceptions in `_run` method in `agent_controller.py`.

So we have to call `sys.exit` (which is ugly but at least it doesn't require changes to `agent_controller` which creates coupling) to exit the test.